### ISSUE TITLE
A11y_code.visualstudio.com_Blog Posts_Strict Null Checking_Screenreader:Unnecessary content is available for the keyboard and screen reader users.

### ISSUE THREAD

**Issue by Charanrajkb** [microsoft/vscode-docs#2809]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com
](url)
Windows Version: Windows10
Browser Version:Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
MAS Violated: MAS 2.1.1, MAS1.3.1

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/blogs/2019/05/23/strict-null](url)
or
Follow below steps:
1. Launch the application Url : [https://code.visualstudio.com
](url)
2. Navigate to "Blog" link.
3. From left navigation, select "Posts".
4. Navigate to "Strict Null Checking" link.
5. Navigate using keyboard Tab and observe the keyboard focus.

**Actual:**
Unnecessary content is available for the keyboard and screen reader users.
While mouse over the keyboard focus goes to the number sign link which is present in front of the all the headings present in the screen and the narrator reads it as number sign link.

**Issue is observed in Edge + Narrator and Chrome + NVDA.**

**Expected:**
Unnecessary content must not be read by the narrator and keyboard users.
The keyboard focus must not go to the number sign link and the keyboard focus must be hidden and the screen reader focus must not go the number sign link which is present after all the headings present in the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
](url)

**User Impact:**
Keyboard users screen readers get confused in understanding the content which is unnecessary.

MAS Reference:
MAS2.1.1:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

MAS1.3.1:
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)

**Attachment For Reference:**
[A11y_code.visualstudio.com_Blog Posts_Strict Null Checking_Keyboard.pptx](https://github.com/microsoft/vscode-docs/files/3325056/A11y_code.visualstudio.com_Blog.Posts_Strict.Null.Checking_Keyboard.pptx)

**Comment by Charanrajkb** [comment]
This issue is observed through out the application.

**Comment by octref** [comment]
Fixed by adding `aria-hidden="true"` to all `#`.
